modules/c++/sys/include/sys/AbstractOS.h

```diff
@@ -187,6 +187,16 @@ class AbstractOS
     virtual std::string getEnv(const std::string& s) const = 0;
 
     /*!
+     *  Returns true if environment variable is set, false otherwise
+     */
+    virtual bool isEnvSet(const std::string& s) const = 0;
+
+    /*!
+     *  Get an environment variable, if set
```

Comment:
Add some details about how returns true/false based on if env variable is set and what `value` is set to in the two cases